maven-core/pom.xml

```diff
@@ -164,11 +164,6 @@ under the License.
       <artifactId>xmlunit-assertj</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-api</artifactId>
-      <scope>test</scope>
-    </dependency>
```

Comment:
Why removing this dependency ?

Reply:
Because is added in root project pom and here we will have a duplicate